js/okcoinusd.js

```diff
@@ -153,6 +153,8 @@ module.exports = class okcoinusd extends Exchange {
                 'price': markets[i]['maxPriceDigit'],
             };
             let lot = Math.pow (10, -precision['amount']);
+            let minAmount = markets[i]['minTradeSize'];
+            let minPrice = 1 / (10 ** precision['price']);
```

Comment:
It cannot transpile the `**`, unfortunately, this is js-only-syntax for now... So, it has to be Math.pow.